### PR TITLE
Neo4-admin-backup-globbing

### PR DESCRIPTION
**Pull request by mauvo**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates.

You can view the updated docs at https://neo4j-docs-operations-1217.surge.sh

**Review by renetapopova — APPROVED** [review]
Looks good to me. The description needs to be updated on the code base as well.

File: modules/ROOT/pages/backup-restore/online-backup.adoc [hunk 1]
```diff
@@ -88,13 +88,13 @@ neo4j-admin database backup [-h] [--expand-commands] [--verbose]
...
 |<database>
-|Name of the remote database to backup. Can contain `*` and `?` for globbing (required unless `--inspect-path` is used).
+|Name of the remote database to backup. Supports globbing inside of double quotes, for example "data*". (<database> is required unless `--inspect-path` is used).
 |neo4j
 |===
 
 [TIP]
 ====
-With a single `*` as a value, you can back up all the databases of the DBMS.
+If <database> is "*" neo4j-admin will attempt to backup all databases of the DBMS.
```
Comment (renetapopova) [inline modules/ROOT/pages/backup-restore/online-backup.adoc:97]:
```suggestion
If <database> is "*", `neo4j-admin` will attempt to back up all databases of the DBMS.
```

File: modules/ROOT/pages/backup-restore/online-backup.adoc [hunk 2]
```diff
@@ -88,13 +88,13 @@ neo4j-admin database backup [-h] [--expand-commands] [--verbose]
 | Default
 
 |<database>
-|Name of the remote database to backup. Can contain `*` and `?` for globbing (required unless `--inspect-path` is used).
+|Name of the remote database to backup. Supports globbing inside of double quotes, for example "data*". (<database> is required unless `--inspect-path` is used).
```
Comment (renetapopova) [inline modules/ROOT/pages/backup-restore/online-backup.adoc:91]:
```suggestion
|Name of the remote database to backup. Supports globbing inside of double quotes, for example, "data*". (<database> is required unless `--inspect-path` is used).
```

**Pull request merged**